users/backends.py

```
@@ -0,0 +1,15 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+
+class EmailBackend(ModelBackend):
+    def authenticate(self, request, username=None, password=None, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `request`

(ARG002)